hr_operating_unit/models/hr_operating_unit.py

```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrOperatingUnit(models.Model):
+    _inherit = "operating.unit"
+
+    employee_ids = fields.Many2many(
+        "hr.employee",
+        "operating_unit_employees_rel",
+        "poid",
```

Comment:
This line was mentioned in the v12 PR that it shouldn't be needed in future versions.

Reply:
Resolve